capi/src/write/get-page.ts

```diff
@@ -45,7 +45,7 @@ export async function getPage(config: t.Config, ctx: t.Ctx): Promise<void> {
 
         writer
           .writeLine("      default: {")
-          .writeLine('        console.error(`No such page "${route}"`);')
+          .writeLine('        console.warn(`No such page "${route}"`);')
```

Comment:
Maybe revert to `error`

Reply:
does it break the page to where a user can not use it?

Reply:
No, it just shows up in the console as an error or a warning.

Reply:
Oh also maybe it blocks the prerendering process if this is an error?  I'll look into it.